### DISCUSSION TITLE
Showcase #1: Vex by nedpals | The V Programming Language

### DISCUSSION THREAD

**Discussion by giscus** [vlang/vlang-blog#12] [General]
# Showcase #1: Vex by nedpals | The V Programming Language

A showcase of the Vex framework that resembles ExpressJS but in written in V.

https://blog.vlang.io/vex-showcase/

**Comment by spytheman** [comment]
If you want to expose your vweb or vex service to the internet, you may also like [The Caddy Server](https://caddyserver.com/docs/quick-starts/https), to handle SSL termination.
With it, exposing a local http server to the internet is very simple, and in most cases boils down to a single line:
```sh
caddy reverse-proxy --from example.com --to localhost:8080
```
